templates/default/src/lib.rs

```diff
@@ -1,58 +1,59 @@
+//! # Concordium V1 Smart Contract Template
+
 #![cfg_attr(not(feature = "std"), no_std)]
 
-//! # A Concordium V1 smart contract
 use concordium_std::*;
 use core::fmt::Debug;
 
-/// Your smart contract state.
+/// The state of the smart contract.
 #[derive(Serialize, SchemaType)]
 pub struct State {
-    // Your state
+    // Add fields to this type to hold state in the smart contract.
 }
 
-/// Your smart contract errors.
+/// Errors that may be emitted by this smart contract.
 #[derive(Debug, PartialEq, Eq, Reject, Serialize, SchemaType)]
 pub enum Error {
     /// Failed parsing the parameter.
     #[from(ParseError)]
     ParseParams,
-    /// Your error
-    YourError,
+    /// Add variants to this enum to be able to return custom errors from the smart contract.
+    CustomError,
 }
 
-/// Init function that creates a new smart contract.
-#[init(contract = "{{crate_name}}")]
+/// Creates a new instance of the smart contract.
+#[init(contract = "{{ crate_name }}")]
 fn init(_ctx: &InitContext, _state_builder: &mut StateBuilder) -> InitResult<State> {
-    // Your code
-
+    // Create the initial state of the smart contract here.
     Ok(State {})
 }
 
-pub type MyInputType = bool;
```

Comment:
It might be useful to add a `CustomInputParameter` to the `init` function and set a very simple state. 

Then we can show in the `deploy_script` how to import the `CustomInputParameter` from the smart contract.

https://github.com/Concordium/concordium-dapp-examples/blob/main/trackAndTrace/test-scripts/Cargo.toml#L13
https://github.com/Concordium/concordium-dapp-examples/blob/main/trackAndTrace/test-scripts/src/main.rs#L9